rotkehlchen/tests/unit/test_data_updates.py

```diff
@@ -172,3 +187,37 @@ def test_update_rpc_nodes(data_updater: RotkiDataUpdater) -> None:
         (11, *custom_node_tuple),
         (12, 'pocket network', 'https://eth-mainnet.gateway.pokt.network/v1/5f3453978e354ab992c4da79', 0, 1, '0.5', 'ETH'),  # noqa: E501
     ]
+
+
+def test_update_contracts(data_updater: RotkiDataUpdater) -> None:
+    """Tests functionality for remotely updating contracts"""
+    with GlobalDBHandler().conn.read_ctx() as cursor:
+        # Check state before the update
+        cursor.execute('SELECT COUNT(*) FROM contract_abi')
+        assert cursor.fetchone()[0] == 73
+        cursor.execute('SELECT COUNT(*) FROM contract_data')
+        assert cursor.fetchone()[0] == 132
```

Comment:
you should state in a comment where those numbers come from. In fact we will have a problem because if in the next update those change due to the contracts being in the globaldb this test will fail. With the assets what we do is ship them already in the packaged globaldb